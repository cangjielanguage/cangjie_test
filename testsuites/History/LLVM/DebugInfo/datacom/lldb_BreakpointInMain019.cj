// DEPENDENCE: ../cjcmd.py ../DatacomTest.py lldb_BreakpointInMain019_expected.txt ../basecase/006_debug_break.cj
// EXEC: cjc -g 006_debug_break.cj
// EXEC: python3 DatacomTest.py lldb_BreakpointInMain019_expected.txt



