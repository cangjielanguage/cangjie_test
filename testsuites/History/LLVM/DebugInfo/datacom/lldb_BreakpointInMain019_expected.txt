lldb,lldb 006_debug_break,006_debug_break
b,b 4,006_debug_break.cj:4
b,b 5,006_debug_break.cj:5
b,b 6,006_debug_break.cj:6
b,b 7,006_debug_break.cj:7
b,b 8,006_debug_break.cj:8
b,b 9,006_debug_break.cj:11
b,b 10,006_debug_break.cj:11
r,r,-> 4
c,c,-> 5
n,n,-> 6
s,s,-> 7
c,c,-> 5
n,n,-> 6
s,s,-> 7
c,c,-> 5
n,n,-> 6
s,s,-> 7
c,c,-> 5
n,n,-> 6
s,s,-> 7
c,c,-> 5
n,n,-> 6
s,s,-> 7
c,c,-> 5
cj_set,cj_set indexbreak = 11,11
n,n,-> 6
s,s,-> 7
n,n,-> 8
n,n,-> 11
cj_plocals,cj_plocals,indexbreak = 12